resources/default_pipeline_environment.yml

```diff
@@ -227,6 +227,7 @@ steps:
     libraryDocumentationUrl: 'https://sap.github.io/jenkins-library/'
     libraryRepositoryUrl: 'https://github.com/SAP/jenkins-library/'
     mandatorySteps: []
+    stepTimeouts: {}
```

Comment:
[minor] just as an idea: wouldn't it be more convenient to define the step times at the level of the steps rather than here on the level of the step `handlePipelineStepErrors`?

Reply:
my goal is to have a central place to maintain it and thus be in synch with the parameter `mandatorySteps` which behaves the same way.